tavla/app/(admin)/tavler/[id]/rediger/components/Buttons/Copy.tsx

```diff
@@ -8,9 +8,24 @@ import { useLink } from 'hooks/useLink'
 function Copy({ type, bid }: { type?: 'button' | 'icon'; bid?: string }) {
     const { addToast } = useToast()
     const link = useLink(bid)
-    const copy = () => {
-        navigator.clipboard.writeText(link ?? '')
-        addToast('Lenken til tavlen ble kopiert!')
+    const copy = async () => {
+        if (!link) {
+            addToast('Lenken er ikke tilgjengelig for kopiering.')
+            return
+        }
+
+        try {
+            await navigator.clipboard.writeText(link)
+            addToast({
+                content: 'Lenken til tavlen ble kopiert!',
+                variant: 'success',
+            })
+        } catch {
+            addToast({
+                content: 'Kunne ikke kopiere lenken. Prøv igjen.',
+                variant: 'information', //TODO: should be error, but the design system have to be bumped up
+            })
+        }
```

Comment:
Vi bør unngå å vise feilmeldinger i en toast, ref. designsystemet her: https://linje.entur.no/komponenter/feedback/alert#universell-utforming. 

> Toast-varsler kan være utfordrende med tanke på universell utforming. Derfor har vi valgt å ikke tilby variantene error og warning. Feilmeldinger og advarsler bør presenteres permanent og i kontekst, nær det aktuelle problemet, med tydelige forslag til rettelser.

Det vi har pleid å bruke er `FormError`. Du kan se et eksempel her: 

```
<FormError {...getFormFeedbackForField('general', state)} />
```
Alle de ulike typene vi har definert for FeedbackError: `app/(admin)/utils/index.ts`